plugins/delpoyment_stats/config/routes.rb

```
@@ -0,0 +1,6 @@
+
+Rails.application.routes.draw do
```

rubocop (1.68.0)

[convention] Unnecessary blank line at the beginning of the source.

(Layout/LeadingEmptyLines)